draft-ietf-ppm-dap.md

```diff
@@ -3077,60 +3271,67 @@ where `batch_interval` is the batch interval requested by the Collector.
 
 ### Batch Buckets {#time-interval-batch-buckets}
 
-Each batch bucket is identified by an interval of time whose beginning is a
-multiple of the task's `time_precision` and whose duration is equal to the
+Each batch bucket is identified by an `Interval` whose duration is equal to the
```

Comment:
Use the `Interval` type so that we implicitly get the definition in terms of `time_precision` as well as the comparison and inclusion semantics.